client/app/components/account_config_main.coffee

```diff
@@ -249,7 +249,7 @@ module.exports = AccountConfigMain = React.createClass
                 # Check for errors before.
                 FormButton
                     class: 'action-check'
-                    spinner: @props.checking
+                    # spinner: @props.checking
```

Comment:
In fact, it was just a lazy comment, but the whole _account_config_main_ file is useless and will be removed :smile: